helm/0.1.0/README.md

```diff
@@ -0,0 +1,225 @@
+
+**Example Installation Instruction for Job Executor**
+```bash
+TASK_SUBSCRIPTION='sh.keptn.event.deployment.triggered\,sh.keptn.event.rollback.triggered\,sh.keptn.event.action.triggered'
+JES_VERSION=0.2.5-next.0
+JES_NAMESPACE=keptn-jes
+
+helm upgrade --install --create-namespace -n ${JES_NAMESPACE} \
+  job-executor-service https://github.com/keptn-contrib/job-executor-service/releases/download/${JES_VERSION}/job-executor-service-${JES_VERSION}.tgz \
+  --set remoteControlPlane.autoDetect.enabled="true",remoteControlPlane.topicSubscription="${TASK_SUBSCRIPTION}",remoteControlPlane.api.token="",remoteControlPlane.api.hostname="",remoteControlPlane.api.protocol=""
+```
+
+**In case you have installed helm-service, please uninstall it**
+```bash
+helm uninstall jmeter-service -n keptn
+```
+
+### Step 2: Make sure your project is setup properly with the right tasks
+
+While technically not part of the installation instructions, it is worthwhile to mention the coupling between Keptn's shipyard file, and the respective Cloud Event Types configured for Job-Executor-Service. 
+
+In example, you should have a monitoring provider (e.g,. Prometheus) configured already for `evaluation` tasks. In addition, for best results, your shipyard should contain at least one, better two stages, a `delivery` sequence, a `rollback` sequence, and a `remediation` sequence.
+
+Example:
+```yaml
+apiVersion: "spec.keptn.sh/0.2.2"
+kind: "Shipyard"
+metadata:
+  name: "shipyard-delivery"
+spec:
+  stages:
+    - name: "qa"
+      sequences:
+        - name: "delivery"
+          tasks:
+            - name: "deployment"
+            - name: "test"
+            - name: "evaluation"
+              properties:
+                timeframe: "2m"
+
+        - name: "rollback"
+          triggeredOn:
+            - event: "qa.delivery.finished"
+              selector:
+                match:
+                  result: "fail"
+          tasks:
+            - name: "rollback"
+
+    - name: "production"
+      sequences:
+        - name: "delivery"
+          triggeredOn:
+            - event: "qa.delivery.finished"
+          tasks:
+            - name: "approval"
+              properties:
+                pass: "manual"
+                warning: "manual"
+            - name: "deployment"
+
+
+        - name: "remediation"
+          triggeredOn:
+            - event: "production.remediation.finished"
+              selector:
+                match:
+                  evaluation.result: "fail"
+          tasks:
+            - name: "get-action"
+            - name: "action"
+            - name: "evaluation"
+              triggeredAfter: "2m"
+              properties:
+                timeframe: "2m"
+
+```
+
+### Step 3: Add the Job-Executor configuration file
+
+The following job-executor configuration file (referred to as job config or `job/config.yaml`) allows you to
+* deploy using `helm upgrade`,
+* rollback using `helm rollback`, and
```

Comment:
```suggestion
* roll back using `helm rollback`, and
```

Reply:
"rollback" is actually a word commonly used in these scenarios (e.g., release management, databases, etc...), so I'd rather keep it :)